README.md

```diff
@@ -1,163 +1,156 @@
-# Connect Applications with Operator-backed Services
+# `service-binding-operator`
```

Comment:
A descriptive title would be nice. Probably it can be combined.
`# service-binding-operator : Connect Applications with Operator-backed Services`

Reply:
Indeed a lengthier tittle would be good, considering the badges centralized in the line below.